src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -622,12 +622,9 @@
 (rf/reg-event-fx :wallet/end-transaction-flow
  (fn [{:keys [db]}]
    (let [address (get-in db [:wallet :current-viewing-account-address])]
-     {:fx [[:dispatch [:wallet/navigate-to-account-within-stack address]]
-           [:dispatch [:wallet/fetch-activities-for-current-account]]
```

Comment:
`:wallet/fetch-activities-for-current-account` was removed here, since it's already called from `:wallet/navigate-to-account-within-stack`